File: HackerRank/ScoreCount.cpp


#include<iostream>
using namespace std;

int main(){

    int n; 
    cin>>n;
    int scores[n];

    //Insert Data, 
    for(int a=0; a<n; a++){
        cin>>scores[a];
    }

    int maximum = scores[0];
    int minimum = scores[0];
    int max  =0;
    int min = 0;

    for(int a=1; a<n; a++){
        if(scores[a]>maximum){
            maximum = scores[a];
            max++;
        }

        if(scores[a]<minimum){
            minimum = scores[a];
            min++;
        }
    }

    cout<<max<<" "<<min<<endl;


    return 0;
}

File: HackerRank/Scorecount.py




class Scorecount:
    def __init__(self) -> None:
        self.score = []
        

    def InputData(self,Data):
        self.score.append(Data)
        



s = Scorecount()

n = int(input())
for x in range(n):
    id = int(input())
    s.InputData(id)

print(s.score)

maximum = s.score[0]
minimum = s.score[0]

max = 0
min = 0

for x in range(n):
    if(s.score[x]>maximum):
        maximum = s.score[x]
        max+=1
    
    if(s.score[x]<minimum):
        minimum = s.score[x]
        min+=1


print(max,min)

